packages/passport/sdk/src/zkEvm/signEjectionTransaction.ts

```diff
@@ -0,0 +1,25 @@
+import { TransactionRequest } from '@ethersproject/providers';
+import {
+  prepareAndSignEjectionTransaction,
+  EjectionTransactionParams,
+  EjectionTransactionResponse,
+} from './transactionHelpers';
+
+type EthSendTransactionEjectionParams = EjectionTransactionParams & {
+  params: Array<any>;
+};
+
+export const signEjectionTransaction = async ({
+  params,
+  ethSigner,
+  zkEvmAddress,
+  flow,
+}: EthSendTransactionEjectionParams): Promise<EjectionTransactionResponse> => {
+  const transactionRequest = params[0] as TransactionRequest;
```

Comment:
@matthewmuscat not sure at which stage of the development cycle we are with these changes, but it'd be nice if we added a pre-condition check to this function call, so check `params` length perhaps.